cypress/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM cypress/included:6.5.0
+FROM cypress/included:6.6.0
```

Comment:
I see there is already version 6.8.0 but maybe we could update that next time to latest.

Reply:
Yes, unless there is any major issue with 6.6.0 we can upgrade next time.